kaggle/src/appsscript.json

```diff
@@ -0,0 +1,22 @@
+{
+  "timeZone": "America/Los_Angeles",
+  "dependencies": {
+    "libraries": [{
+      "userSymbol": "FirebaseApp",
+      "libraryId": "1hguuh4Zx72XVC1Zldm_vTtcUUKUA6iBUOoGnJUWLfqDWx5WlOJHqYkrt",
+      "version": "28"
+    }]
+  },
+  "exceptionLogging": "STACKDRIVER",
+  "dataStudio": {
+    "name": "Kaggle",
+    "logoUrl": "https://www.kaggle.com/content/v/43edd17f0e8c/kaggle/img/logos/kaggle-logo-transparent.png",
+    "company": "vlookup@google.com",
+    "companyUrl": "https://developers.google.com/datastudio/",
+    "addonUrl": "https://developers.google.com/datastudio/",
+    "supportUrl": "https://developers.google.com/datastudio/",
+    "description": "Kaggle datasets in Data Studio. Please use they API key as password.",
```

Comment:
Please make the first sentence a complete sentence that describes what this connector does. Also, s/they/the/